Library/Formula/jena.rb

```diff
@@ -1,8 +1,8 @@
 class Jena < Formula
   desc "Framework for building semantic web and linked data apps"
   homepage "https://jena.apache.org/"
-  url "https://archive.apache.org/dist/jena/binaries/apache-jena-2.12.1.tar.gz"
-  sha256 "5bbda78a873c1f27af1719e1d1e724a56a0739d90a9450fc3b4b710ad128734b"
+  url "http://archive.apache.org/dist/jena/binaries/apache-jena-3.0.1.tar.gz"
```

Comment:
`* Stable: Please use https:// for http://archive.apache.org/dist/jena/binaries/apache-jena-3.0.1.tar.gz`